src/ui/Controllers/OrganisationController.cs

```diff
@@ -82,19 +112,36 @@ public async Task<IActionResult> About(string ucasCode)
         [HttpPost]
         [Route("{ucasCode}/about")]
         public async Task<ActionResult> AboutPost(string ucasCode, OrganisationViewModel model)
+        {
+            var ucasInstitutionEnrichmentGetModel = await _manageApi.GetEnrichmentOrganisation(ucasCode);
+            var result = await SaveOrgansation(ucasInstitutionEnrichmentGetModel, model, ucasCode);
```

Comment:
nb. in a future version where we have a contactdetails editor, this would override the contact details fields with empties